strings/newspaper.txt

```diff
@@ -0,0 +1,39 @@
+publisher
+Nanotrasen Weekly@,
+The Daily Bulletin@,
+The Mostly Independent@,
+The Nanotrasen Times@,
+Headline Daily
+
+headline
+John Nanotrasen casually admits to committing multiple felonies in leaked meeting recordings.@,
+Study by PSFCSO shows presence of Owls directly correlated with station longevity.@,
+Recent survey shows over 80% of NT employees admit to having died at least once on the job.@,
+New Nadir Extraction Site called 'a bit damp' by disgruntled Head Of Personnel.@,
+Atlas discovered to be bee shaped. 'Yeah, we designed it that way?' say architects.@,
+Xiang Giesel sued by same person for the 6th time. 'Just make the engines safer' says engineer.@,
+Silicon Union destroyed by singularity bombs. NT claims Union was front for Syndicate.@,
+Rogue artificial intelligence talked down by simply saying 'nuh uh' repeatedly.@,
+Retraction: We apologise for calling Fugere 'Šid' and vice versa for 13 years straight.@,
+Chaplain shot for accidentally holding bible upside down in funeral.@,
+Nuclear Engine onboard NT-13 melts down. Again.@,
+Questions raised about effectiveness of flash weapons.@,
+Man found dead in NT Centcomm offices turned out to be faking it.@,
+Hafgan smartphone released. Critics call it 'overpriced PDA'.@,
+Super!Friend sued for releasing 4 million amusing ducks into local ecosystem.@,
+Thinktronic announces that Guardbuddies will not be upgraded. 'They're perfect as is' states CEO.@,
+Enraged MD goes on a monkey killing spree, upsetting janitors.@,
+Genetek releases new cloning pod. Critics unable to find any new features.@,
+Zvezda launches new publicity stunt. 14 dead, 18 wounded. CEO calls it 'smashing success'.@,
+Space Diner once again unlivable.@,
+GardenGear proposes new joint project with NT.@,
+NT Space Station 17 voted 'worst station' for 9th year running.@,
+Calls for more railings around trench holes.@,
+Oshan tourism at all time highs.@,
+Highest peak on Mundus climbed for first time. Climber says 'hardest part was actually getting there'.@,
+Pug detective sniffs out crimes.
+
+article
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Mollis aliquam ut porttitor leo a diam sollicitudin tempor. Libero justo laoreet sit amet cursus. Nam at lectus urna duis convallis convallis tellus id interdum. Donec pretium vulputate sapien nec sagittis. Duis tristique sollicitudin nibh sit amet commodo. Ut sem nulla pharetra diam sit amet nisl suscipit adipiscing. Sed odio morbi quis commodo. Nulla aliquet porttitor lacus luctus accumsan tortor posuere ac ut. In egestas erat imperdiet sed euismod nisi porta lorem.@,
+Introduction. Todo: actually write this damn article.@,
+Once upon a time, there was a very nosy person. They opened a newspaper and decided to read the news. That person is you. Mind your own business.
```

Comment:
Feels like there should be more of these? Maybe use the CFG stuff to generate them rather than writing out each one.

Reply:
damn i completely forgot about these. Yeah I'll add some more. What's CFG?

Reply:
Context free grammar, basically the system used to build up new categories from previous ones like `[pronoun] [verb]ed a [noun]`